src/category_theory/adjunction/basic.lean

```diff
@@ -297,9 +354,20 @@ end equivalence
 
 namespace functor
 
+/-- An equivalence `E` is left adjoint to its inverse. -/
 def adjunction (E : C ⥤ D) [is_equivalence E] : E ⊣ E.inv :=
 (E.as_equivalence).to_adjunction
 
+/-- If `F` is an equivalence, it's a left adjoint. -/
+instance left_adjoint_of_equivalence {F : C ⥤ D} [is_equivalence F] : is_left_adjoint F :=
```

Comment:
The linter correctly advises setting this and the next instance as lower priority, so that "structural" instances have a chance to fire first.